src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioAnalyzerTests.vb

```diff
@@ -42,33 +35,6 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
             End Using
         End Sub
 
-        <WpfFact>
-        Public Sub AnalyzerErrorsAreUpdated()
-            Using workspace = New TestWorkspace(composition:=s_compositionWithMockDiagnosticUpdateSourceRegistrationService)
-                Dim lazyWorkspace = New Lazy(Of VisualStudioWorkspace)(
-                                        Function()
-                                            Return Nothing
-                                        End Function)
-
-                Dim file = Path.GetTempFileName()
-
-                Dim hostDiagnosticUpdateSource = New HostDiagnosticUpdateSource(lazyWorkspace)
-
-                Dim eventHandler = New EventHandlers(file)
-                AddHandler hostDiagnosticUpdateSource.DiagnosticsUpdated, AddressOf eventHandler.DiagnosticAddedTest
```

Comment:
this was the only location that .DiagnosticsUpdated was referenced.  removing this meant all the above code could be removed.